src/master-playlist-controller.js

```diff
@@ -172,6 +172,20 @@ export default class MasterPlaylistController extends videojs.EventTarget {
     this.masterPlaylistLoader_.load();
   }
 
+
+  getMediaRequests_() {
```

Comment:
Naming nitpick (sorry!): video.js doesn't typically use the `get` prefix for getters. That is, `s/getMediaRequests/mediaRequests/g`.